proto/kvrpcpb.proto

```diff
@@ -655,3 +655,140 @@ message ReadIndexResponse{
     uint64 read_index = 2;
 }
 
+enum VerOp {
+    VerPut = 0;
+    VerDel = 1;
+}
+
+message VerValue {
+    VerOp op = 1;
+    bytes value = 2;
+    uint64 version = 3;
+}
+
+message VerKvPair {
+    KeyError error = 1;
+    bytes key = 2;
+    repeated VerValue values = 3;
+}
+
+message VerGetRequest {
+    Context context = 1;
+    bytes key = 2;
+    string cf = 3; 
+    int32 version_num = 4;
+}
+
+message VerGetResponse {
+    errorpb.Error region_error = 1;
+    string error = 2;
+    repeated VerValue value = 3;
+    bool not_found = 4;
+}
+
+message VerBatchGetRequest {
+    Context context = 1;
+    repeated bytes key = 2;
+    string cf = 3;
+    int32 version_num = 4;
+}
+
+message VerBatchGetResponse {
+    errorpb.Error region_error = 1;
+    repeated VerKvPair pairs = 2;
+}
+
+message VerGetWithSVRequest {
```

Comment:
I think we can just merge it to `VerGetRequest`. `start_version` == 0 means without version

Reply:
fixed it. PLAT again.